src/Shared/UnitTests/FileMatcher_Tests.cs

```diff
@@ -558,6 +565,11 @@ public void IllegalPaths()
         [Fact]
         public void IllegalTooLongPath()
         {
+            if (NativeMethodsShared.IsUnixLike)
+            {
+                // Nothing's too long for Unix
```

Comment:
Is this true in our current implementation?  I guess that's just a hole in our preemptive `MAX_PATH` limits, which is just as well since we'll need to remove them entirely.